tests/test_methods_and_attributes.py

```diff
@@ -261,7 +261,7 @@ def test_property_rvalue_policy():
 
 
 # https://foss.heptapod.net/pypy/pypy/-/issues/2447
-@pytest.mark.xfail("env.PYPY")
+@pytest.mark.xfail("env.PYPY", strict=True)
```

Comment:
Why are we adding `strict=True` instead of using `xfail_strict=True` in `pytest.ini`?
I'm probably missing something simple.

Reply:
I have a test where I want `strict=False`, because it seems to randomly fail. And I didn't realize that the ini option only changes the default (oops!).